x-pack/plugins/observability_solution/observability_onboarding/e2e/cypress/e2e/logs/custom_logs/configure.cy.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 // Failing: See https://github.com/elastic/kibana/issues/186215
-describe.skip('[Logs onboarding] Custom logs - configure step', () => {
+describe('[Logs onboarding] Custom logs - configure step', () => {
```

Comment:
I could not make this to fail, I suspect some other change already has "fixed" it.